apisix/admin/init.lua

```diff
@@ -199,7 +199,17 @@ local function run()
     end
 
     local code, data
-    if seg_res == "routes" then
+    local refactored_resources = {
```

Comment:
directly named as `resources`?

Reply:
There has been a global variable named `resources` 😭. Maybe we can rename that variable after all resources are refactored.

Reply:
ok